.github/workflows/build.yml

```diff
@@ -26,7 +25,7 @@ jobs:
       - name: Build wheels
         env:
           CIBW_BUILD: "cp${{ matrix.python-version }}-*"
-          CIBW_SKIP: "*-manylinux_i686 *-win32"
+          CIBW_SKIP: "*-manylinux_i686 *-win32 *-macosx_arm64"
```

Comment:
Is this intentional? We still want arm64 wheels for macOS right?

Reply:
Yes, added it back to be keep it same as before